Svc/Framer/docs/sdd.md

```diff
@@ -20,6 +20,12 @@ in each frame; typically it is a frame header, a data packet, and a hash value.
 You can use the standard F Prime downlink protocol implementation.
 This implementation works with the F Prime ground data system (GDS).
 
+`Svc::Framer` is designed to act alongside instances of the
+[communication adapter interface](https://nasa.github.io/fprime/Design/communications-adapter-interface.html) and thus
```

Comment:
I see a couple of issues with this explanation:

1. It refers to the communication adapter interface page, but the diagram there has the Communication Adapter sending status to the Com Queue. It looks like the intent of the new design is to have the Communication Adapter send status to the Framer, which forwards the status to the Com Queue. We should update the diagram to match the new design.
2. It says that the Framer must not violate the communication adapter protocol. However, the protocol applies to communication adapters, and Framer is not a communication adapter. I think we need to revise the protocol. For example, we could rename it to Communication Adapter Status protocol and say that it must be followed by any Communication Adapter or any component that sends data to and forwards status from a Communication Adapter.

Reply:
For item 2, maybe it's better to add a new Framer protocol governing the way the framer sends packets and sends back status. I think the Framer protocol itself is nontrivial and should be described. For example:

1. It seems the intent is that if n packets are aggregated into 1 frame, Framer sends back n - 1 success status messages, then sends out a frame, and ComAdapter sends back a status message which is forwarded by Framer, for a total of n status messages. This procedure should be spelled out (if it is correct).
2. What if Framer receives one packet and emits n frames? Do n status messages go back in this case? Is ComQueue equipped to handle n status messages in response to a single packet?

Reply:
Or in item 2 above, should Framer wait for n status messages from ComAdapter and forward one message back to ComQueue? This behavior would be more symmetrical.